packages/slick-components/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@vtex/slick-components",
+  "version": "1.0.1",
```

Comment:
1.0.0 or 1.0.1?

Reply:
Cause everytime that I run the lerna publish it automatically changes the version

Reply:
👍